great_expectations/data_context/types/base.py

```diff
@@ -94,7 +94,7 @@ def _get_schema_instance(cls) -> Schema:
             return cls.get_config_class().schema_instance
 
     @classmethod
-    def from_commented_map(cls, commented_map: CommentedMap):
+    def from_commented_map(cls, commented_map: CommentedMap):  # type: ignore[no-untyped-def]
```

Comment:
Why are we doing this? Don't we know the type of the return?

Reply:
@cdkini this was from before -- I just added the `ignore` macro.